generate/seccomp/doc.go

```diff
@@ -1,2 +0,0 @@
-// Package seccomp includes functions for configuring seccomp.
-package seccomp
```

Comment:
I believe it was was just a place holder that Haiyan added to tell me where the seccomp library should go when she separated out generate. I can't think of anything more to say, so I don't think we gain anything from leaving it. It's also just a single line....If we want to add it back with more information then we're not being set back by removing it now.